### DISCUSSION TITLE
Is it possible to drag the connection from the output node to reassign it from another output?

### DISCUSSION THREAD

**Discussion by OnurEAkgul** [Foblex/f-flow#77] [Q&A] [answered]
I couldn't find a new FOutputId from the FReassignConnectionEvent. And I can't drag the connection output from the output node. i can only drag from the input node.

**Comment by siarheihuzarevich** [comment, marked as answer]
@OnurEAkgul Hello, you can only drag from input to another input

**Reply by OnurEAkgul** [reply]
Ok, I have found a way to get ahead of this with changing the output id to another output id, will you be adding this feature in the future? I can open an issue for a feature request if you want

**Reply by siarheihuzarevich** [reply]
Yes, but not soon)
